functional-tests/src/test/java/com/redhat/che/functional/tests/Che7DialogAbout.java

```diff
@@ -108,25 +108,25 @@ public void openDialogAbout() {
   }
 
   private TestWorkspace createChe7Workspace() {
-    prepareWorkspace("che7", WORKSPACE_NAME, null);
+    prepareWorkspace("Java Maven", WORKSPACE_NAME, null);
     projectSourcePage.clickOnAddOrImportProjectButton();
     newWorkspace.clickOnCreateButtonAndOpenInIDE();
     return testWorkspaceProvider.getWorkspace(WORKSPACE_NAME, defaultTestUser);
   }
 
-  private void prepareWorkspace(String stack, String workspaceName, Double machineRam) {
+  private void prepareWorkspace(String stackID, String workspaceName, Double machineRam) {
     dashboard.waitDashboardToolbarTitle();
     LOG.info("Opening Workspaces from left menu...");
     dashboard.selectWorkspacesItemOnDashboard();
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
     LOG.info("Selecting stack...");
-    selectStack();
+    selectStack(stackID);
```

Comment:
do we really want to operate with stacks in Che 7 ?

Reply:
the name is misleading, It's really the element ID of a devfile.

Reply:
@ScrewTSW than we need a better method naming ;-)

Reply:
I will refactor it :)